### ISSUE TITLE
Mouse wheel scrolling

### ISSUE THREAD

**Issue by mikeott** [taitems/jQuery.Gantt#204]
Great work on this BTW. I'm going to implement it into an existing project.

One small issue though.

If you're using your mouse wheel to scroll down the page, as soon as the Gantt chart hits your mouse pointer the page scrolling action is suddenly hijacked.

I realise the Gantt chart will let you scroll left/right in the timeline with your mouse wheel and that's the reason for the hijacking, but from a usability POV this is unexpected behaviour (and kind of frustrating). I also think it a little nonsensical that moving your mouse wheel up and down should make something move left and right.

If possible it would be good if there was a way to disable this behaviour, perhaps a new parameter called `mouseWheelScroll` or similar with options `true` or `false`, where `false` would let the user scroll down the page past the Gantt chart without interruption.

![mouse-wheel-scrolling](https://cloud.githubusercontent.com/assets/5625353/16403724/b3a7d070-3d2a-11e6-9684-69c48fec4405.gif)

Thanks in advance.

Keep up the good work.

**Comment by gregthegee** [comment]
Hi I'm new to jQuery Gantt. scroll was not working.

**Comment by usmonster** [comment]
Hi @gregthegee! Thanks for commenting, but can you please create a new issue and elaborate on what you mean by "not working". Thanks!

**Comment by mikeott** [comment]
Agreed with everything you said, and thanks for the tip. I had to comment out the mousewheel event as well.
